surveys/sphinx-survey-2025.yml

```diff
@@ -215,20 +217,6 @@ questions:
       - "PDF"
       - "EPUB"
 
-  - title: "How comfortable do you feel when working with docutils?"
-    choices:
-      - "Very comfortable"
-      - "Comfortable"
-      - "Neutral"
-      - "Uncomfortable"
-      - "Very uncomfortable"
-      - "I don't know or don't work with docutils"
```

Comment:
For personal use, some of this information would be interesting, though we probably have better contact with extension developers than 'ordinary' users. It might be worth thinking about a conditional section behind "do you develop a sphinx extension"? This is somewhat marginal though, so if a less complex survey is wanted then I'm content with this.

Reply:
I know there was discussion about why this was here and if it fit with the larger goals, so I thought it might be easiest to remove right now. I do know @Carreau has mentioned having more certainty that things will work right with less complex survey logic, so that is my goal right now too.

Reply:
Another of my fear is better is the enemy of good; 

I thing one of our goal should be to run survey regularly, learn from results and iterate, and the more we try to be perfect the more we burnout people wanting to:

1) work on the survey
2) take the survey
3) analyze the survey

I also think that if there are a subset of users (say extension developers), we can at the end have a "are you an extension developers, we want to know more, here is another survey"

That way we still have not taken them too much time and "Trap" them in a longer survey.